### DISCUSSION TITLE
Unavailable Diffusions

### DISCUSSION THREAD

**Discussion by rtaskf** [RupertAvery/DiffusionToolkit#231] [Q&A]
So, the program looks very useful and all, but I moved a whole bunch of 1000+ images to my archive, out of the scanned folder, and I can't seem to get rid of all the "unavailable" items now. I can't possibly right-click and remove them all. Refreshing or rebuilding meta data doesn't help and I can't even find a filter for unavailable. How am I supposed to clean up the data-base now?

**Comment by rtaskf** [comment]
Ok well, I just deleted the whole database file now. First I tried those 3 options under "Tools->Folders", which sounded very much like they could fix the issue, but didn't really do anything after all.

Anyway, if there is another fix, besides deleting the database file, I'd be interested to know.

**Comment by rtaskf** [comment]
Well, in any case. It's a very nice tool. Thank you! I've got thousands of gens that I can finally filter and sort easily, and drag right from the app into A1111's png info to further work with.

**Comment by RupertAvery** [comment]
The latest commits has **Tool > Unavailable Files** for dealing with this for now. 

This build artifact should  have these changes, if you don't want to wait for an official build:

https://github.com/RupertAvery/DiffusionToolkit/actions/runs/8773444541/artifacts/1433316835

Latest builds will appear in https://github.com/RupertAvery/DiffusionToolkit/actions. Click on the workflow run and scroll down to the artifacts and download DiffusionToolkit.
